esp/eclwatch/ws_XSLT/batchworkunits.xslt

```diff
@@ -378,7 +378,7 @@
                             <td><b>Batch Workunits</b>
                             </td>
                             <td>
-                                <input type="checkbox"  id="AutoRefresh" value="Auto Refresh" onclick="setAutoRefresh(this);"/> Auto Refresh
+                                <label><input type="checkbox"  id="AutoRefresh" value="Auto Refresh" onclick="setAutoRefresh(this);"/>Auto Refresh</label>
```

Comment:
Is it appropriate to remove the space? Most inputs don't include spaces, but some include non-breaking spaces and a couple include regular spaces.

Reply:
Yes, it is empty. This is another way to label a checkbox:
   <label><input type="checkbox" ... /> Label Text </label>